documentation/src/main/wiki/ActionKnot.md

```diff
@@ -12,7 +12,7 @@ successful / error / next step page.
 Let's describe Action Knot behaviour with following example.
 
 ### Example
-ActionKnot processes only those fragments that `<script>` tag has defined `data-api-type="form-{NAME}"` parameter, 
+ActionKnot processes only those fragments which `data-knot-types` parameter contains "form-{NAME}", 
```

Comment:
ActionKnot processes fragments having "form-{NAME}" in `data-knot-types` attribute, ....